ConsentViewController/Classes/SourcePointClient/SourcePointClient.swift

```diff
@@ -451,4 +452,45 @@ extension SourcePointClient {
             })
         }
     }
+
+    func metaDataURLWithParams(
+        env: SPCampaignEnv,
+        accountId: Int,
+        propertyId: Int,
+        metadata: ConsentStatusMetaData
```

Comment:
@Nevazhnovu the `metadata` of the `/meta-data` request is different from the `metadata` from the `/consent-status` request :D 

I believe you'll need to implement another struct to encapsulate it. Its attributes should be:
```
{
    ccpa?: {
        hasLocalData: boolean
        dateCreated?: string
        uuid?: string
    },
    gdpr?: {
        hasLocalData: boolean
        dateCreated?: string
        uuid?: string
    }
}
```
You can base your new struct on the `ConsentStatusMetaData`.